runtime/crab/src/lib.rs

```diff
@@ -136,7 +138,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: sp_runtime::create_runtime_str!("Crab"),
 	impl_name: sp_runtime::create_runtime_str!("Darwinia Crab"),
 	authoring_version: 0,
-	spec_version: 42,
+	spec_version: 43,
```

Comment:
Not a good idea to bump spec version within this PR

Reply:
Yes,  the version change should be included in a separated pr. 

I am not sure which branch should be used when testing the DVM features in the pangolin network. 

1. Merge this pr into master firstly and testing use master branch
2. Directly use this pr branch to do test work, merge it after the test ends.

I prefer the 1. Test using master and add fix if something is broken.

Thus, The bump version change should be revert here. I will do it later after the locally test successfully.

Reply:
1 for final test (but not this PR)
2 for local test